cmd/node/main.go

```diff
@@ -86,62 +86,70 @@ VERSION:
 
 	// genesisFile defines a flag for the path of the bootstrapping file.
 	genesisFile = cli.StringFlag{
-		Name:  "genesis-file",
-		Usage: "The node will extract bootstrapping info from the genesis.json",
+		Name: "genesis-file",
+		Usage: "The `filepath` for the genesis file in JSON format. This file should contain initial data to bootstrap" +
+			" from, such as initial balances for accounts.",
 		Value: "./config/genesis.json",
 	}
 	// nodesFile defines a flag for the path of the initial nodes file.
 	nodesFile = cli.StringFlag{
-		Name:  "nodes-setup-file",
-		Usage: "The node will extract initial nodes info from the nodesSetup.json",
+		Name: "nodes-setup-file",
+		Usage: "The `filepath` for the nodes setup in JSON format. This file should contain initial nodes info, such as" +
```

Comment:
The path to the Nodes setup. This JSON file contains initial info about Nodes, such as ...